src/theme.tsx

```diff
@@ -31,6 +31,20 @@ const theme = {
       xl: "h-12 w-12",
     },
   },
+  badge: {
+    base: "font-sans font-medium text-xs rounded-full",
+    variants: {
+      primary: "text-white bg-gray-800",
+      secondary: "text-white bg-blue-500",
+      outline: "bg-white text-gray-800 border border-gray-800",
+    },
+    size: {
+      xs: "text-xs px-2 py-1",
```

Comment:
Add this `py-1` to base !

```suggestion
      xs: "text-xs px-2",
```